docs/scos/dev/migration-program/migration-to-paas/paas-migration-documents/integrate-the-latest-version-of-docker-sdk-and-run-project-on-it.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Integrate the latest version of Docker SDK and run project on it
+description: This document describes how to integrate the latest version of Docker SDK and run project on it.
+template: howto-guide-template
+---
+
+# Integrate the latest version of Docker SDK and run project on it
+
+{% info_block infoBox %}
+
+## Resources for assessment Backend, DevOps[optional]
+
+{% endinfo_block %}
+
+1. Create boot files for all required environments using the following pattern `deploy.(projectname)-(environmentname).yml`
+    and the following [example](https://github.com/spryker-shop/b2c-demo-shop/blob/master/deploy.spryker-b2c-staging.yml).
+2. Boot local development environment based on newly created deploy file and ensure that the local env is working well.
+3. In sync with the cloud engineer check the naming of hosts for services per each environment.
+4. Deploy other environments in Spryker Cloud and ensure they are running well with new/updated boot files.
+   The first deployment of any environment has to be in destructive mode.
+5. Custom applications have to be added by DevOps, the following instructions are needed in case it is not part of Spryker applications:
```

Comment:
Do I need to provide the following information to a devops to add a custom application?